content/cosign-3-0-available.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "cosign v3 is now available"
+author = "Zach Steindler (GitHub)"
+date = "2025-10-08"
+tags = ["sigstore", "cosign", "opensource"]
+type = "post"
+draft = false
++++
+
+The past few years have been incredible in the Sigstore ecosystem, seeing Sigstore-signed in-toto attestations be adopted by [Homebrew](https://blog.trailofbits.com/2024/05/14/a-peek-into-build-provenance-for-homebrew/) (May 2024), [PyPI](https://blog.pypi.org/posts/2024-11-14-pypi-now-supports-digital-attestations/) (November 2024), [Maven Central](https://central.sonatype.org/news/20250128_sigstore_signature_validation_via_portal/) (January 2025), and model signing in [NVIDIA's NGC](https://developer.nvidia.com/blog/bringing-verifiable-trust-to-ai-models-model-signing-in-ngc/) (July 2025).
```

Comment:
Do you want to call out npm and RubyGems as well?

Reply:
There's also Kaggle where the model signing work started - I'm going to throw "and others" on here instead of trying to exhaustively list everything.